### PR TITLE
ADEN-3796 AdEngine runner timeout

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/ARecoveryEngine/js/sourcePointRecovery.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*global define*/
+define('ext.wikia.aRecoveryEngine.recovery.sourcePointRecovery', [
```
Comment (nandy-andy) [inline extensions/wikia/ARecoveryEngine/js/sourcePointRecovery.js:2]: Don't we need in this module implementation of `getPrices()` as well? 

edit:
It's being used in `ext.wikia.adEngine.lookup.lookupFactory` where the `ext.wikia.aRecoveryEngine.recovery.sourcePointRecovery` module isn't being used as lookup module, so probably we don't but I decided it was worth asking ;)

**Comment by nandy-andy** [comment]
Two things:
- merge `dev` branch, or rebase your branch to see if it helps to fail unit tests (they've been failing in `dev` and it's just got fixed a few hours ago,
- `ext.wikia.aRecoveryEngine.recovery.sourcePointRecovery` is being used like lookup module but it's not and it's not totally clear -- feel free to chat with me offline or let's think if it's the right path and if it doesn't need refactoring (if it does then separate ticket should be fine).

File: extensions/wikia/ARecoveryEngine/js/sourcePointRecovery.js (hunk 1 above)
Comment (fraszczakszymon) [inline extensions/wikia/ARecoveryEngine/js/sourcePointRecovery.js:2]: Exactly, it's not used as a bidder :)

**Comment by fraszczakszymon** [comment]
@nandy-andy 
ad. second bullet - it's not used exactly like lookup module. It's used only in AdEngineRunner - that's why I renamed all 'bidders' to 'modules'

**Comment by nandy-andy** [comment]
@fraszczakszymon sure, thanks for the clarification! I think I was just biased because of those lines ;)

```
require.optional('ext.wikia.adEngine.lookup.amazonMatch'),
require.optional('ext.wikia.adEngine.lookup.rubiconFastlane'),
require.optional('ext.wikia.aRecoveryEngine.recovery.sourcePointRecovery')
```

and some similar functions in the new module. Maybe then the delaying part of lookups should be separated to make it all consistent and clean.

Well, that's a longer discussion not for her, I guess...

LGTM :japanese_ogre:

**Pull request merged**